workers/unity/Packages/com.improbable.gdk.mobile/Editor/MobileLaunchConfig.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections.Generic;
+using UnityEditor;
+
+namespace Improbable.Gdk.Mobile
+{
+    public class MobileLaunchConfig
+    {
+        public bool shouldConnectLocally;
+        public string RuntimeIp => EditorPrefs.GetString(RuntimeIpEditorPrefKey);
+        public string DevelopmentTeamId => EditorPrefs.GetString(DevelopmentTeamIdEditorPrefKey);
+        
+        internal Dictionary<string, string> availableSimulators = new Dictionary<string, string>();
+        internal Dictionary<string, string> availableDevices = new Dictionary<string, string>();
+
+        internal string DevelopmentTeamIdEditorPrefKey = "DevelopmentTeam";
+        internal string RuntimeIpEditorPrefKey = "RuntimeIp";
```

Comment:
don't we define this key somewhere else as well?

Reply:
yeah, still need to clean this up a bit. this key should most likely be defined in the tools package , then used here and not being exposed in the gdk tools config.